src/exercises/ExerciseTypeConstants.js

```diff
@@ -12,10 +12,7 @@ export const EXERCISE_TYPES = {
   findWordInContext: "Recognize_L1W_in_L2T",
 
   isTranslationExercise: function (exerciseType) {
-    return (
-      exerciseType === this.translateWhatYouHear ||
-      exerciseType === this.multipleChoice
-    );
+    return exerciseType === this.multipleChoice;
```

Comment:
why is `translateWhatYouHear` not a translation exercise anymore?

Reply:
Excellent question! This actually doesn't do anything – all exercises work as intended without it. It never worked, since we are not passing the exercise type from the multiple choice exercise to the `TranslatableText` component. I've fixed that now and pushed all changes to this branch.